src/game/Spells/SpellEffects.cpp

```diff
@@ -697,15 +697,6 @@ void Spell::EffectDummy(SpellEffectIndex eff_idx)
                     }
                     return;
                 }
-                case 26471: // Lunar Festival Port Error
-                {
-                    if (!m_caster->IsPlayer())
-                        return;
-
-                    m_caster->RemoveSpellCooldown(26373); // Remove cooldown from Lunar Invititation
```

Comment:
why, this is needed to not set cooldown when u use it in wrong area

Reply:
its wrong.. no one sniffed it before me, it just teleports you to the moonglade position

Reply:
it was just a quick improvisation on me, because nobody could tell me how it really works